### ISSUE TITLE
Is there a way to get a common cut-off value?

### ISSUE THREAD

**Issue by KimHyeonKi** [volkamerlab/teachopencadd#114]
"T007 · Ligand-based screening: machine learning" was very helpful for us. Thanks for sharing this article.

I just have one question.

We are using IC50 data targeting other cells of ChEMBL, but our data does not have a few comments to divide "active" and "inactive".

Therefore, we also have to divide based on the pIC50 value, and I would like to know how to get the cut-off of 6.3 in the article you shared above.

I will wait for your reply.

Thank you again.

- Kim Hyeon Ki

**Comment by dominiquesydow** [comment]
Hi @KimHyeonKi,

Thanks for posting this issue!

We based the pIC50 cutoff of 6.3 on this work:
https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01611 
https://github.com/Team-SKI/Publications/blob/master/Profiling_prediction_of_kinase_inhibitors/Build_ABL1_model.ipynb

Cutoff do differ in literature though, e.g. here a pIC50 of 6.5 is used:
https://pubs.acs.org/doi/10.1021/ci500342q
https://jcheminf.biomedcentral.com/articles/10.1186/s13321-017-0232-0

My colleague Lindsey Burggraaff commented on the pitfalls defining an active class in this review:
https://pubs.acs.org/doi/10.1021/acs.jcim.8b00832

I hope this helps!

**Comment by KimHyeonKi** [comment]
@dominiquesydow 

Thank you for your kind reply.

We looked at the articles you recommended and came to the conclusion that the cut-off value of a lot of data for each target is flexible and should be decided in consultation with the person who analyzes the data and the person who does the modeling.

We collected the various toxicity thresholds mentioned in each assay through the source of the IC50 data obtained from ChEMBL, and decided to take a cut-off value to the extent that the class imbalance of the data does not become severe.

In addition, we considered a method to use after excluding some data at the boundary of the cut-off value so as not to significantly affect the number of existing training data. However, since we do not know how much data loss this method will cause, we plan to apply it in the future.

Thanks again for your help.

- KimHyeonKi
